### PR TITLE
Add Carl on libbitcoinkernel

### PR DESCRIPTION
**Pull request by michaelfolkson**

### CODE REVIEW

File: carl-dong/_index.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+title: Carl Dong
```
Comment (adamjonas) [inline carl-dong/_index.md:2]: I'm -0.5 on introducing a new top-level index for this. Carl already has 4 talks and so creating a new folder doesn't really seem like it makes sense. Given the UI on the index page, I think it's worth being stingy with creating these.

Comment (michaelfolkson) [reply carl-dong/_index.md:2]: You mean put it an existing directory (say the chaincode-labs directory) and not create a new one for Carl?

Either is fine. No personal views on the matter whatsoever.

Comment (michaelfolkson) [reply carl-dong/_index.md]: Done the above, I think that's what you meant anyway

**Comment by michaelfolkson** [comment]
Ready for re-review

**Pull request merged**